Python/README.md

```diff
@@ -0,0 +1,156 @@
+# HQIU Python Coding Style
+
+This document provides an outline of HQIU's coding style in Python.
+
+## Purpose
+
+This Python style guide helps contribute to the goal of the Healthcare Quality Intelligence unit (HQIU) in producing reproducible and transparent code. Adhering to rules outlined in this style guide helps to improve our ability to share, maintain, and develop analytical workflows in cases where Python is used. 
+
+This is designed for those in the HQIU team or those interested in writing Python code that is clear and can be shared and maintained. The HQIU Python coding style guide takes inspiration from the HQIU R coding style guide.
+
+## Principles
+
+In accordance with the HQIU R style guide.
+
+* We take a disciplined and practical approach to writing code.
+* We regularly check-in code to Github
+* We believe consistency in style is very important.
+* We demonstrate intent explicitly in code, via clear structure and comments where needed.
+
+## Rules
+
+The basis of the style guide is a combination of principles from PEP 8 and the tidyverse style guide, which is a document that provides the guidelines and best practices on writing python code. Using PEP 8 helps to improve the readability of Python Code.
```

Comment:
Add link to PEP 8. What are the conflicts of tidyverse style guide and PEP 8? I'm happy to move to PEP 8 if there are inbuilt stylers/linters.

Reply:
I have added a suggested formatter and linter. As PEP8 is a guide, not all of the linters and formatters follow it exactly. I have identified 2 options which are both simple, and do not seem to conflict with the tidyverse style guide. Pylint for example checks for snake case, unused variables and unused imports